charts/synthetics-job-manager/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: synthetics-job-manager
 description: New Relic Synthetics Containerized Job Manager
 type: application
-version: 1.0.13
+version: 1.0.14
```

Comment:
This will need to be updated to version 1.0.15 since we [released a 1.0.14 already](https://github.com/newrelic/helm-charts/pull/991)